section.css

```diff
@@ -0,0 +1,58 @@
+
+.Section {
+	margin: 0 auto;
+	text-align: center;
+}
+
+.Section--background{
```

Comment:
Remember that ".Section--background" would be a modifier on ".Section", and at the block level. This is probably more of an element of ".Section", so ".Section__background".